run_dosbox

```diff
@@ -110,12 +111,14 @@ def run(cmd_line, wait=False):
             run_dosbox(['-conf', confgen.create_audio_conf(), exe, '-exit'])
         else:
             run_dosbox(['-conf', confgen.create_audio_conf(),
-                        '-c', f'@echo File not found: {os.getcwd()}/{exe}'])
+                        '-c', '@echo File not found: {0}/{1}'.format(
+                            os.getcwd(), exe)])
```

Comment:
Hmm, after using format this no longer fits neatly and it's hard to tell what is a parameter and what is a list element at first glance…

Rewrite it this way, should be a bit more readable:

            exe_path = os.path.join(os.getcwd(), exe)
            run_dosbox(['-conf', confgen.create_audio_conf(),
                        '-c', '@echo File not found: ' + exe_path)

Reply:
It's actually possible to do this:
```
            exe_path = os.path.join(os.getcwd(), exe)                                                                                                                                                             
            run_dosbox(['-conf', confgen.create_audio_conf(),                                                                                                                                                     
                        '-c', '@echo File not found: {0}'.format(exe_path)])     
```
while avoiding using `+` to concatenate strings, and still come out no longer in line length than the `run_dosbox()` call just above it.

Reply:
Both are fine :)